### DISCUSSION TITLE
No highlight set for DiffChange?

### DISCUSSION THREAD

**Discussion by bertin0** [olimorris/onedarkpro.nvim#185] [Q&A] [answered]
I think it would be a good idea to set it, as leaving it unset (at least on neovim) leaves the default magenta bg for changed lines when doing a diff is quite jarring and doesn't match. I see that there used to be a DiffChange variable in the colors object but it's commented out?

**Comment by olimorris** [comment]
Oooof. Good spot. Can you share a screenshot?

**Comment by bertin0** [comment]
![image](https://github.com/olimorris/onedarkpro.nvim/assets/30109573/f64343ea-32ad-4241-91c0-59ab57ee8cd2)

**Reply by olimorris** [reply, marked as answer]
Can you check the latest commit? I think setting the background to nothing is the way to go as per the comment in #186
